### DISCUSSION TITLE
Alternative for transform.setFromMatrix?

### DISCUSSION THREAD

**Discussion by arkus-pm** [pixijs/pixijs#9742] [v8 General]
I'm using matrices from my internal model and has been assigning it via
```
container.transform.setFromMatrix(
      new PIXI.Matrix(...myModel.transformMatrix.values)
    )
```

Is there any alternative for this? Seems like layerTransform/localTransform/worldTransform.copyFrom method don't do anything

**Comment by Nicox-gogo** [comment]
Does anyone know? I have the same question.

**Comment by GoodBoyDigital** [comment]
hey there! is this to set a local matrix?
